### PR TITLE
Typo-fix

### PR DESCRIPTION
**Pull request by SingingApple**
## Related to typo
Fixed a typo in **gettingstarted.stories.mdx**

## Description
Changed _compoents_ to _components_ in **gettingstarted.stories.mdx** at line 157.

## Checklist
 
- [ ] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Meta } from '@storybook/addon-docs/blocks'
+import { Meta } from "@storybook/addon-docs/blocks";
```
Comment (nimishbongale) [inline packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx]: If you have a look at the other meta stories for Vocabulary, you'll notice we use single quotes for the import paths.

File: packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx [hunk 2]
```diff
@@ -154,7 +167,7 @@ We use a Storybook addon called [storybook-addon/docs](https://storybook.js.org/
 
 ### Step 4 - Choosing Components
 
-The blog site looks a bit empty right now. Maybe we should add in a few more compoents!
+The blog site looks a bit empty right now. Maybe we should add in a few more components!
```
Comment (nimishbongale) [inline packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx:170]: Good find! 👍

File: packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx [hunk 3]
```diff
@@ -175,7 +188,14 @@ Footer (mandatory)
...
 
-<iframe width="100%" height="600px" src="//jsfiddle.net/nimishbongale/7b94ozac/1/embedded/html,result/dark/" allowFullScreen="allowfullscreen" allowpaymentrequest frameBorder="0"></iframe>
+<iframe
+  width="100%"
+  height="600px"
+  src="//jsfiddle.net/nimishbongale/7b94ozac/1/embedded/html,result/dark/"
+  allowFullScreen="allowfullscreen"
+  allowpaymentrequest
+  frameBorder="0"
+></iframe>
```
Comment (nimishbongale) [inline packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx:198]: We may switch this out for CodePen prefill embeds (#837) but thanks for this anyways!

**Review by nimishbongale — COMMENTED** [review]
Thanks for this! 👍 

There are a lot more changes than just fixing the typo (some dealing with code formatting), so that needs to be taken into account as well.

Suggested a few changes, please have a look!

File: packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx (hunk 1 above)
Comment (SingingApple) [reply packages/vocabulary/.storybook/meta/gettingstarted.stories.mdx]: I've resolved this in the newer commit.

**Review by nimishbongale — APPROVED** [review]
LGTM 👍

**Review by zackkrida — CHANGES_REQUESTED** [review]
Thank you @SingingApple; while you're here could you spell check this entire page `gettingstarted.stories.mdx`? I noticed `personal` is spelled as `peronal` in a few places too.

**Comment by nimishbongale** [comment]
> I noticed `personal` is spelled as `peronal` in a few places too.

Whops! Fixed it 😄

**Comment by SingingApple** [comment]
@zackkrida I'll have a look and will make the changes if I find some by tomorrow.

**Comment by SingingApple** [comment]
@zackkrida Couldn't find any typos on this page!

**Review by zackkrida — APPROVED** [review]
Thank you! FYI, your pull request contained _formatting changes_ to a few files, likely caused by prettier + vscode automatically formatting the files you opened. If you are working on a lot of open source projects I would recommend _this_ vscode setting:

**Pull request merged**